service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,7 +80,11 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
         Document document = uploadDocumentService.uploadPDF(userId, authorization, pdf, buildFileName(caseDetails));
 
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        sdf.setTimeZone(TimeZone.getTimeZone("Europe/London"));
+
         Map<String, Object> data = caseDetails.getData();
+        data.put("dateSubmitted", sdf.format(new Date(System.currentTimeMillis())));
```

Comment:
I'd more expect use of new java.time API e.g. `DateTimeFormatter`

Reply:
👍